### DISCUSSION TITLE
Welcome to pyjoulescope_server Discussions!

### DISCUSSION THREAD

**Discussion by mliberty1** [jetperch/pyjoulescope_server#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by earsnot** [comment]
Hi :)
My name is Anders Rasmussen, I am a danish engineer developing smart water meters.
We are using the Joulescope for our in the loop CI/CD setup for different metrics.
I am currently implementing a client in .NET C# to interact with the pyjoulescope server.
